xml/ha_storage_protection.xml

```diff
@@ -332,6 +332,12 @@
         is considered delivered. The timeout should be long enough for the node to
         detect that it needs to self-fence.
        </para>
+       <para>
+        However, if the <literal>msgwait</literal> timeout is too long, it
+        can be possible that a fenced cluster node rejoins before the
```

Comment:
[...] is too long, it is possible [...] -> [...] is too long, a fenced cluster node might rejoin [...]
Maybe add the following sentence at the end of this para: "This can be mitigated by setting the SBD_DELAY_START parameter in the SBD configuration, as described in LINK TO PROCEDURE OR STEP".

Reply:
Ok, here is first attempt:

![msgwait-timeout](https://user-images.githubusercontent.com/1312925/39306798-6463583c-4961-11e8-80d6-81ff552a4b72.png)